modules/ppcp-api-client/src/Entity/FraudProcessorResponse.php

```diff
@@ -51,10 +51,10 @@ public function avs_code(): ?string {
 	/**
 	 * Returns the CVV response code.
 	 *
-	 * @return string|null
+	 * @return string
 	 */
 	public function cvv_code(): ?string {
```

Comment:
I think this can't be `null` now, because you have `$this->cvv2_code = (string) $cvv2_code;` in constructor and `(string) null` will retrun an empty string, so the return value of this method is always a string like in Dockblock comment

Reply:
Yes but since you have cast in constructor already
```
$this->avs_code  = (string) $avs_code;
$this->cvv2_code = (string) $cvv2_code;
```

you have already forced it to be only a string. Sorry, feel free please to ignore

Reply:
Ah now I have seen what you mean ;) I will change it. Thanks!